sdk/python/feast/loaders/file.py

```diff
@@ -76,10 +78,10 @@ def export_source_to_staging_location(
                 os.path.join(source_uri.netloc, source_uri.path)
             )
         else:
-            # gs, s3 file provided as a source.
+            # gs, s3, azure blob file provided as a source.
             assert source_uri.hostname is not None
             return get_staging_client(source_uri.scheme).list_files(
```

Comment:
seems like the Config object isn't available to be passed to get_staging_client here, so is there a better way to pass in the required parameters (account name and access key) for the Azure client than requiring them to be passed in in the config?